sentry/src/main/java/io/sentry/MonitorContexts.java

```diff
@@ -0,0 +1,90 @@
+package io.sentry;
+
+import io.sentry.util.Objects;
+import io.sentry.vendor.gson.stream.JsonToken;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class MonitorContexts extends ConcurrentHashMap<String, Object>
+    implements JsonSerializable {
+  private static final long serialVersionUID = 3987329379811822556L;
```

Comment:
Do you even need this field, as the class isn't `@Serializable`?

Reply:
Just copied this over. I'll remove it.

Reply:
Ah it's `ConcurrentHashMap` that's `Serializable` so I'll add it back.

Reply:
Ahh, my bad - I missed that - sorry and thanks for re-adding!